game.py

```diff
@@ -20,8 +20,34 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses  = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
 
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES  :
+
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+
+        if user_input in snowman_word :
+            print("You guessed a letter that's in the word!")
+            correct_letter_guess_statuses [user_input] =True
+
+        else:
+            print(f"The letter {user_input} is not in the word")
+            wrong_guesses_list.append(user_input) 
+            
+        print_snowman_graphic(len(wrong_guesses_list))
+
+        print_word_progress_string(snowman_word,correct_letter_guess_statuses)
+
+        if is_word_guessed(snowman_word, correct_letter_guess_statuses) :
+            print("\n")
+            print (f"Congratulations, you win!")
+            return None
```

Comment:
A return statement in Python stops the execution of code, which is why I suspect you return `None` here. I'm wondering if you can think of another way the logic could be written so that you can end the game without needing to return `None` to break the loop?

This isn't a requirement, but if you'd like more practice with writing conditional logic in Python, I'd suggest thinking through and re-writing the logic so you don't need return to stop the game.

If you choose to do so, there's no need for you to submit anything to me for re-review. This would just be for your own practice 😊